Dynatrace/dynatrace-operator
# Add helm package generation to GitHub actions

## Description

This PR add the ability to generate helm packages directly in the release action. 

In addition the manifest generation was refactored such that we don't use a matrix anymore but rather a for loop.

## How can this be tested?

Create a tag, run these changes in GitHub actions and check if new files were appended to draft release

## Checklist
